sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubSchemaCapableIOProvider.java

```diff
@@ -264,4 +265,22 @@ private static boolean fieldPresent(
               schema.getField(field).getType(), Schema.EquivalenceNullablePolicy.IGNORE);
     }
   }
+
+  @Internal
+  @AutoValue
+  public abstract static class Config implements Serializable {
```

Comment:
Can we make this class private or package-private?

Reply:
Done